spec/enumerables_spec.rb

```diff
@@ -0,0 +1,194 @@
+require './enumerables.rb'
```

Comment:
Use `require_relative` instead. Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.